cartography/intel/aws/organizations.py

```diff
@@ -86,6 +91,55 @@ def get_aws_accounts_from_botocore_config(boto3_session: boto3.session.Session)
     return d
 
 
+def describe_organization(boto3_session: boto3.session.Session) -> Dict:
```

Comment:
The convention is to name this with `get`, can we do something like this instead?

```suggestion
def get_organizations(boto3_session: boto3.session.Session) -> Dict:
```